docs/getting-started/development-prerequisites.md

```diff
@@ -2,7 +2,7 @@
 
 Writing an iTwin.js application requires the following software:
 
-- [Node.js](https://nodejs.org) (latest 12.x LTS version)
+- [Node.js](https://nodejs.org) (latest 14.x LTS version)

```

Comment:
Should probably be explicit here and elsewhere that newer versions (current is 16.x) are **not** supported - users ran into issues with this.

Reply:
I was hoping the "LTS" would cover that case.  Maybe just a link to the download would be best?

Reply:
Hmm, yeah, LTS ought to be sufficient. The user in question had lots of trouble following other instructions.